include/odp/api/spec/classification.h

```diff
@@ -55,6 +55,11 @@ extern "C" {
  * @deprecated Use ODP_PMR_INVALID instead
  */
 
+/**
+ * Default classification mark value.
+ */
+#define ODP_CLS_MARK_DEFAULT 0
```

Comment:
Do we really need this? I thought we would just say that the default value is zero and that is it.

Reply:
OK. I will remove this.